app/services/subscriber_list_params/generate_links_based_list_params_service.rb

```diff
@@ -1,51 +1,53 @@
-class GenerateSubscriberListParamsService < ApplicationService
-  def initialize(content_item)
-    super()
-    @content_item = content_item
-  end
+module SubscriberListParams
+  class GenerateLinksBasedListParamsService < ApplicationService
+    def initialize(content_item)
+      super()
+      @content_item = content_item
+    end
 
-  def call
-    {
-      "title" => content_item["title"],
-      "links" => link_hash,
-      "url" => content_item["base_path"],
-    }
-  end
+    def call
+      {
+        "title" => content_item["title"],
+        "links" => link_hash,
+        "url" => content_item["base_path"],
+      }
+    end
 
-private
+  private
 
-  attr_reader :content_item
+    attr_reader :content_item
 
-  class UnsupportedContentItemError < StandardError; end
+    class UnsupportedContentItemError < StandardError; end
```

Comment:
Just semantic, but you should move this out of the private part of this class given it's part of the public interface.

Reply:
Thanks, have made that change.